bootstrap/bootstrap_linux.bash

```diff
@@ -38,6 +37,11 @@ if [ -d $machin_dir ]; then
     echo "Entering machine dir: $machine_dir"
     cd $machine_dir
 fi
+machine_dir=$HOME/machines/$HOSTNAME
+if [ -d $machin_dir ]; then
```

Comment:
`if [ -d $machine_dir ]; then`
(an "e" is missing)

Reply:
You are correct. Fixed this as well as the two other places where this error apparently have existed for a long while. Wondering how it ever worked...